src/routes/Authentication.js

```diff
@@ -4,7 +4,80 @@ import { validateUserInput } from '../middleware/middleware';
 
 const indexRouter = express.Router();
 
+/**
+ * @swagger
+ * /signup:
+ *    post:
+ *      summary: Create a new User.
+ *      requestBody:
+ *        required: true
+ *      responses:
+ *        201:
+ *          description: A list of users
```

Comment:
What is returned is not a list of users, it's the object of the newly created user.